src/sdt.rs

```diff
@@ -117,6 +118,14 @@ where
          * and length, and then the dispatched to the correct function to actually parse the table.
          */
         match signature {
+            "FADT" => {
+                let fadt_mapping = handler.map_physical_region::<Fadt>(physical_address);
+                match parse_fadt(&fadt_mapping) {
```

Comment:
This could just be `parse_fadt(&fadt_mapping)?;`

Reply:
Yep :wink: